help/sources/tutorials/ui/create/crm/sugarcrm-events.md

```diff
@@ -0,0 +1,91 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Create a SugarCRM Events source connection in the UI 
+description: Learn how to create a SugarCRM Events source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Create a [!DNL SugarCRM Events] source connection in the UI
+
+This tutorial provides steps for creating a [!DNL SugarCRM Events] source connector using the Platform user interface.
+
+## Overview
+
+[[!DNL SugarCRM]](https://www.sugarcrm.com/) is a customer relationship management (CRM) system. [!DNL SugarCRM]'s functionality includes sales-force automation, marketing campaigns, customer support, collaboration, Mobile CRM, Social CRM and reporting.
+
+To retrieve event data setup within [!DNL SugarCRM], this Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) connector leverages the [!DNL SugarCRM] [Events API](https://market.apidocs.sugarcrm.com/#516ec3b1-8e70-43d4-8bf2-38a2ae74c0a5).
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Events] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Events API.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Events] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
+
+### Gather required credentials
+
+In order to connect [!DNL SugarCRM Events] to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| *`Host`* | The SugarCRM API host. | `developer.salesfusion.com` |
+| *`Username`* | Your SugarCRM developer account username. | `abc.def@example.com@sugarmarketdemo000.com` |
+| *`Password`* | Your SugarCRM developer account password. | `123456789` |
+
+### Create Platform Schema
+
+Create a Platform [schema](/help/xdm/schema/composition.md) required for [!DNL SugarCRM Events].
+
+![Platform UI screenshot showing an example schema for SugarCRM Events](../../../../images/tutorials/create/sugarcrm-events/sugarcrm-schema-events.png?lang=en)
+
+>[!WARNING]
+> When mapping the schema ensure you also map the mandatory event_id and timestamp fields required by Platform.
```

Comment:
Need to add another ">" in between here, and also need to remove the space between ">" and the text, so:
>[!WARNING]
>
>When mapping...